sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/aio/_schema_registry_client_async.py

```diff
@@ -136,19 +138,36 @@ async def register_schema(
             content_type=kwargs.pop(
                 "content_type", "application/json; serialization={}".format(format)
             ),
-            **http_request_kwargs
+            **http_request_kwargs,
         )
 
         response = await self._generated_client.send_request(request, **kwargs)
         response.raise_for_status()
         return _parse_response_schema_properties(response, format)
 
+    @overload
     async def get_schema(self, schema_id: str, **kwargs: Any) -> Schema:
-        """
-        Gets a registered schema by its unique ID.
-        Azure Schema Registry guarantees that ID is unique within a namespace.
+        ...
+
+    @overload
+    async def get_schema(
+        self, *, group_name: str, name: str, version: int, **kwargs
+    ) -> Schema:
+        ...
+
+    @distributed_trace_async
+    async def get_schema(self, *args: Union[str, int], **kwargs: Any) -> Schema:
```

Comment:
why are the args a Union of str and int? Can you pass schema Id or version as args?

Reply:
ah good catch! I think args should just be str - will fix this. I added int here b/c version is an int. However, version will be passed in as a kwarg(`version=`) and not a positional argument, so we don't need the type of version to be included in args.

So schema_id can be passed in as a positional argument, (i.e. we don't require the `schema_id=` to be passed in) or as a keyword arg (i.e. schema_id=schema_id). If it's passed in as a positional arg, it'll be in `args`. If it's passed in as a kwarg, then the type will be included in `Any`.